src/custom/api/ipfs/index.ts

```diff
@@ -1,8 +1,35 @@
-import pinataSDK, { PinataPinResponse } from '@pinata/sdk'
 import { PINATA_API_KEY, PINATA_SECRET_API_KEY } from 'constants/ipfs'
 
-const pinata = pinataSDK(PINATA_API_KEY, PINATA_SECRET_API_KEY)
+export async function pinJSONToIPFS(file: any): Promise<any> {
```

Comment:
why did you change the return type? 

returning any doesn't look to good

Reply:
My idea was if I import that type I also need to import the library but I will just add the type in our code